spec/lib/public_cocina_record_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe PublicCocinaRecord do
+  let(:druid) { 'bb021mm7809' }
+  let(:purl_url) { 'https://purl.stanford.edu' }
+  let(:public_cocina_record) { described_class.new(druid, purl_url:) }
+
+  def stub_purl_request(druid, body)
+    stub_request(:get, "https://purl.stanford.edu/#{druid}.json").to_return(status: 200, body:)
+  end
+
+  before do
+    stub_purl_request(druid, File.read(file_fixture("#{druid}.json").to_s))
```

Comment:
can you extract `let(:body) { File.new(file_fixture("#{druid}.json") }` (you don't need to read the file).